tests/foreman/installer/test_installer.py

```diff
@@ -1446,6 +1484,34 @@ def test_installer_options_and_flags():
     assert PREVIOUS_INSTALLER_OPTIONS == current_installer_options, msg
 
 
+@pytest.mark.upgrade
+@pytest.mark.tier3
+def test_installer_help_sections():
+    """Look for changes on installer sections
```

Comment:
No need to add a separate test case here most of the things are common, just add a parameterized fixtures to verify all params, sections, and subsections.

```
@pytest.mark.upgrade
@pytest.mark.tier3
@pytest.mark.parametrize("type", ['params', "sections", "subsections"])
def test_installer_options_and_flags(type):
```

Reply:
Thanks for the nice suggestion here. Implemented !